### ISSUE TITLE
[Bug]: Gaining infinite runes when i enter in friend world

### ISSUE THREAD

**Issue by raibm** [LukeYui/EldenRingSeamlessCoopRelease#479]
### Discord Username

Passfailed#9394

### Describe the issue in detail

I and my friends were playing normaly for the last month, but i was every time the host, and they just join in my world. Last week we had a problem to join in my world, so we decide to join in my friend world. But he killed some bosses out of mod, so when i entered i got all his runes (around 100k), and if i disconect and join again, it heppened again, all the times. If i do this five times, i`ll gain 500k runes, for example.

### Steps to recreate the bug

1 - Create a character in your world and kill some bosses, so use the item called Tiny Great Pot
2 - Invite a player to your world (he need to use the item Effigy of Malenia and he cant kill the "killed boss" in your world too)
3 - See the magic happen

### Version

1.4.3 (Latest)

### Relevant Crash log

```shell
Seamless_Co-op_BETA_1.4.2-510-1-4-2-1672180222.zip
669.77 KB
```

**Comment by Aerron03** [comment]
I've acted as the host each time and have heard of my joining partner have a similar issue. It doesn't seem to happen every time, but sometimes when joining the session he gets a "great enemy felled" and is granted tens of thousands of runes. The amount doesn't seem to be consistent either.

Happens on 1.4.3

**Comment by Chawbacon** [comment]
This has happened twice now with people joining on me as well. Free soul inflation is kind of OP.

**Comment by H4wK32** [comment]
I can reproduce it.

**Comment by minutemaen** [comment]
Started happening to us as well.

All started with Flying dragon Agheel being defeated, then joining player reconnecting, upon connection he gets great enemy felled and the souls from killing Agheel.
Then we killed Leonine Misbegotten and had the player rejoin, he got the same enemy felled message and the souls for only Leonine

**Comment by LukeYui** [comment]
Fixed in 1.5.0
